CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+## Contributing
+
+[fork]: https://github.com/probot/probot/fork
+[pr]: https://github.com/probot/probot/compare
+[style]: https://github.com/probot/eslint-config-probot
+[code-of-conduct]: CODE_OF_CONDUCT.md
+
+Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+
+Please note that this project is released with a [Contributor Code of Conduct][code-of-conduct]. By participating in this project you agree to abide by its terms.
+
+## Submitting a pull request
+
+0. [Fork][fork] and clone the repository
+0. Configure and install the dependencies: `npm install`
+0. Make sure the tests pass on your machine: `npm test`, note: these tests also apply the linter, so no need to lint seperately
+0. Create a new branch: `git checkout -b my-branch-name`
+0. Make your change, add tests, and make sure the tests still pass
+0. Push to your fork and [submit a pull request][pr]
+0. Pat your self on the back and wait for your pull request to be reviewed and merged.
```

Comment:
The `0.` causes the numbering to start at 0 now instead of 1 due to a recent (in the last ~3 months) change in markdown formatting.